wg-multitenancy/annual-report-2021.md

```diff
@@ -5,56 +5,57 @@
+   - Hierarchical Namespace Controller
+   - Virtual Cluster Project
+   - Multi-tenancy Benchmark Project
+   - Multi-tenancy Documentation Project
 
 ## Project health
 
 1. What's the current roadmap until completion of the working group?
 
-   -
-   -
-   -
+   - We had a healthy discussion in December 2021 about whether it was time to spin the working group down or move it to another SIG, or petition for it to become its own SIG, given the long lived nature of its work. From that discussion the leaders of the WG agreed that our next obvious milestone is to add robust documentation to the Kubernetes' website on how to achieve multi-tenancy. We consistently get asked about how to do this, and the upstream documentation has no docs. So we agreed to pick this up as our next major milestone, and are executing on it. We also had some new projects start to emerge, so we are staying open minded about whether we should continue to provide a space for new projects, or if we should wind down once our documentation project is complete.
 
 2. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, we have leaders from VMware, Google, Alibaba, Nirmata, and Twilio. We also have a revolving series of contributors from many different companies in addition to these.
 
 3. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - No, I think we're well supported with attention and concrete work at this time. 
+   
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
+- Primary slack channel member count: 1168
+- Primary mailing list member count: 738
+- Primary meeting attendee count (estimated, if needed): 10
+- Primary meeting participant count (estimated, if needed): 10
 
 Include any other ways you measure group membership
 
 ## Operational
 
 Operational tasks in [wg-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
-- [ ] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
-- [ ] Updates provided to sponsoring SIGs in 2021
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      -
+- [ x] [README.md] reviewed for accuracy and updated if needed
+- [ x] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
+- [ x] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
+- [ x] Updates provided to sponsoring SIGs in 2021
```

Comment:
links to the mailing list threads or meetings or recordings where this happened would be great for folks interested in the summaries